### PR TITLE
Updating A6263 and A6264 Deploy CSVs

### PR DESCRIPTION
**Pull request by cdobs**
Made the following updates:
-Corrected DOSTA SN for 06263 D9, D10, and D11 to 1520
-Corrected DOSTA SN for 06264 D11, D12, D13, and D14 to 1519
-Uncommented lines for 06263 D1 and D5
-Uncommented lines for 06264 D5
-Updated deploy and recover CUIDs

### CODE REVIEW

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 1]
```diff
@@ -48,31 +48,31 @@ WJ20190228,,WJ20190228,,CP05MOAS-A6263-03-CTDAVN000,7,1,2019-03-01T07:53:00,2019
...
+AR34,,AR34,,CP05MOAS-A6263-06-PARADN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-00-ENG000000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.203167,-71.09655,,,,along-shelf survey
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-01-FLORTN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.203167,-71.09655,,,,along-shelf survey
-WJ20190617,,WJ20190617,,CP05MOAS-A6263-02-DOSTAN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,40.203167,-71.09655,,,,along-shelf survey
+WJ20190617,,WJ20190617,,CP05MOAS-A6263-02-DOSTAN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01520,40.203167,-71.09655,,,,along-shelf survey
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6263_Deploy.csv:60]: Looks good.  Associated checkout document .zip file will need to be revised - remove the 1519 cal doc and insert the 1520 cal doc. (3403-40001-00017-A.zip)

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 2]
```diff
@@ -48,31 +48,31 @@ WJ20190228,,WJ20190228,,CP05MOAS-A6263-03-CTDAVN000,7,1,2019-03-01T07:53:00,2019
...
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-05-ADCPAN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.203167,-71.09655,,,,along-shelf survey
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-06-PARADN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.203167,-71.09655,,,,along-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6263-00-ENG000000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.01075,-70.567783,,,,along-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6263-01-FLORTN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.01075,-70.567783,,,,along-shelf survey
-WJ20190709,,WJ20190709,,CP05MOAS-A6263-02-DOSTAN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-02-DOSTAN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01520,40.01075,-70.567783,,,,along-shelf survey
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6263_Deploy.csv:67]: Looks good.  Associated checkout document .zip file will need to be revised - remove the 1519 cal doc and insert the 1520 cal doc. (3403-40001-00019-A.zip).  [zip file also still references previous cruise P12b]

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 3]
```diff
@@ -48,31 +48,31 @@ WJ20190228,,WJ20190228,,CP05MOAS-A6263-03-CTDAVN000,7,1,2019-03-01T07:53:00,2019
...
-AR-39,,AR-39,,CP05MOAS-A6263-05-ADCPAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
-AR-39,,AR-39,,CP05MOAS-A6263-06-PARADN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR39,,AR39,,CP05MOAS-A6263-00-ENG000000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR39,,AR39,,CP05MOAS-A6263-01-FLORTN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR39,,AR39,,CP05MOAS-A6263-02-DOSTAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01520,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6263_Deploy.csv:74]: Looks good. Associated checkout document .zip file will need to be revised - remove the 1519 cal doc and insert the 1520 cal doc. (should be 3403-40001-00021-A.zip but zip file does not exist in Vault, just the associated checkout test)

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 4]
```diff
@@ -1,99 +1,99 @@
...
+AR34,,AR34,,CP05MOAS-A6264-06-PARADN000,10,1,2019-04-10T17:00:00,2019-04-11T17:07:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.91693,-70.906,,,,across-shelf survey
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-00-ENG000000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-01-FLORTN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
-WJ20190617,,WJ20190617,,CP05MOAS-A6264-02-DOSTAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-02-DOSTAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:74]: Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (3403-40001-00018-A.zip)

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 5]
```diff
@@ -1,99 +1,99 @@
...
 WJ20190617,,WJ20190617,,CP05MOAS-A6264-06-PARADN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.93057,-70.90167,,,,abbreviated mission due to time constraints (AC-1 to AC-2)
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-00-ENG000000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-01-FLORTN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,40.34183,-70.89003,,,,across-shelf survey
-WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,40.34183,-70.89003,,,,across-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,40.34183,-70.89003,,,,across-shelf survey
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:81]: Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (3403-40001-00020-A.zip) [zip file also still references previous cruise P12b]

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 6]
```diff
@@ -1,99 +1,99 @@
...
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-06-PARADN000,12,1,2019-07-10T09:00:00,2019-07-11T07:01:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,40.34183,-70.89003,,,,across-shelf survey
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-00-ENG000000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-01-FLORTN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
-WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
+WJ20190709,,WJ20190709,,CP05MOAS-A6264-02-DOSTAN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:88]: Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (3403-40001-00020-A.zip) [zip file also still references previous cruise P12b]

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 7]
```diff
@@ -1,99 +1,99 @@
...
-AR-39,,AR-39,,CP05MOAS-A6264-05-ADCPAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-06-PARADN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-00-ENG000000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-01-FLORTN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-02-DOSTAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:95]: Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (should be 3403-40001-00022-A.zip but zip file does not exist in Vault, just the associated checkout test)

**Review by rtravis-whoi — APPROVED** [review]
The checkout procedure testing .zip files need to be updated in Vault for the DOSTA sensor switches:
AUV 06263 - 3403-40001-00017, -00019, -00021 (see individual comments for each deployment where the DOSTA SN was updated) *there isn't a zip file for -00021
AUV 06264 - 3403-40001-00018, -00020, -00022 (see individual comments for each deployment where the DOSTA SN was updated) *there isn't a zip file for -00022

**Review by dbogorff — APPROVED** [review]

**Pull request merged**